TAF/utils/scripts/docker/deploy-edgex.sh

```diff
@@ -49,6 +44,6 @@ fi
 docker run --rm -v ${WORK_DIR}:${WORK_DIR}:rw,z -w ${WORK_DIR} -v /var/run/docker.sock:/var/run/docker.sock \
         --env WORK_DIR=${WORK_DIR} --env PROFILE=${PROFILE} --security-opt label:disable \
         --env DS_PROFILE=${DS_PROFILE} --env CONF_DIR=${CONF_DIR} --env REGISTRY_URL=${REGISTRY_URL} \
-        ${COMPOSE_IMAGE} -f "${WORK_DIR}/TAF/utils/scripts/docker/docker-compose${BACKWARD}.yaml" up -d ${DATABASE} consul data metadata command notifications scheduler
+        ${COMPOSE_IMAGE} -f "${WORK_DIR}/TAF/utils/scripts/docker/docker-compose${BACKWARD}.yaml" up -d redis consul data metadata command notifications scheduler
```

Comment:
Since [the change](https://github.com/edgexfoundry/developer-scripts/pull/291/commits/d9417a35dd8343788f46b814d2b649e867895fd9), the "redis" should be changed to "database".